protocol/x/clob/keeper/msg_server_place_order.go

```diff
@@ -36,76 +36,82 @@ func (k msgServer) PlaceOrder(goCtx context.Context, msg *types.MsgPlaceOrder) (
 // 1. persisting the placement on chain.
 // 2. updating ProcessProposerMatchesEvents with the new stateful order placement.
 // 3. adding order placement on-chain indexer event.
+// Various logs, metrics, and validations are skipped for orders internal to the protocol.
 func (k Keeper) HandleMsgPlaceOrder(
 	ctx sdk.Context,
 	msg *types.MsgPlaceOrder,
+	isInternalOrder bool,
 ) (err error) {
 	lib.AssertDeliverTxMode(ctx)
 
-	// Attach various logging tags relative to this request. These should be static with no changes.
-	ctx = log.AddPersistentTagsToLogger(ctx,
-		log.Module, log.Clob,
-		log.ProposerConsAddress, sdk.ConsAddress(ctx.BlockHeader().ProposerAddress),
-		log.Callback, lib.TxMode(ctx),
-		log.BlockHeight, ctx.BlockHeight(),
-		log.Handler, log.PlaceOrder,
-		log.Msg, msg,
-	)
-
-	defer func() {
-		metrics.IncrSuccessOrErrorCounter(
-			err,
-			types.ModuleName,
-			metrics.PlaceOrder,
-			metrics.DeliverTx,
-			msg.Order.GetOrderLabels()...,
+	if !isInternalOrder {
+		// Attach various logging tags relative to this request. These should be static with no changes.
+		ctx = log.AddPersistentTagsToLogger(ctx,
+			log.Module, log.Clob,
+			log.ProposerConsAddress, sdk.ConsAddress(ctx.BlockHeader().ProposerAddress),
+			log.Callback, lib.TxMode(ctx),
+			log.BlockHeight, ctx.BlockHeight(),
+			log.Handler, log.PlaceOrder,
+			log.Msg, msg,
```

Comment:
let's also keep the persistent tags at the for the initial version. in case we have issues this helps us debug

can skip the defered func below